lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java

```diff
@@ -0,0 +1,5 @@
+package com.lightstep.tracer.jre;
+
+class Version {
+    static final String LIGHTSTEP_TRACER_VERSION = "0.12.5";
```

Comment:
ah got it, I forget we talked about the versions moving independently of common now, thanks!